packages/shared/src/components/layout/MainLayoutHeader.tsx

```diff
@@ -150,6 +153,10 @@ function MainLayoutHeader({
     );
   };
 
+  if ((isMobile || isTablet) && useNewMobileLayout) {
```

Comment:
I think instead of `isMobile || isTablet` we can use !isLaptop

Reply:
Also I see the useNewMobileLayout already excludes isLaptop, and in this case looks like we don't need this part of the condition at all `(isMobile || isTablet)`